R/tbl_propdiff.R

```diff
@@ -28,22 +28,28 @@
 #'
 #' @section Methods:
 #'
-#' The `exact`` option gives the exact confidence interval for the unadjusted
-#' difference in proportions as calculated by `prop.test`.
+#' * The `chisq` option returns a p-value from the `prop.test` function and a
+#' confidence interval for the unadjusted difference in proportions based on
+#' the normal approximation.
 #'
-#' The `boot_centile` option calculates the adjusted difference between groups
+#' * The `exact` option returns a p-value from the `fisher.test` function. The
+#' confidence interval returned by this option is the same as the confidence
+#' interval returned by the `chisq` option and is based on the normal approximation.
+#'
+#' * The `boot_centile` option calculates the adjusted difference between groups
 #' in all bootstrap samples (the default for this method is 2000 resamples)
 #' and generates the confidence intervals from the distribution of these
 #' differences. For the default, a 95% confidence interval, the 2.5 and 97.5
 #' centiles are used.
 #'
-#' The `boot_sd` option calculates the adjusted difference between groups
+#' * The `boot_sd` option calculates the adjusted difference between groups
 #' in all bootstrap samples (the default for this method is 200 resamples).
 #' The mean and standard deviation of the adjusted difference across all
 #' resamples are calculated. The standard deviation is then used as the
 #' standard error to calculate the confidence interval based on the true adjusted difference.
 #'
 #' @examples
+#' library(gtsummary)
```

Comment:
Is this needed?  If so, why?

Reply:
I don't think it is now - I think this was from before when the whole gtsummary package wasn't loaded so that we could use the "trial" dataset in the example. I've taken it out.